tools/modules/EnsEMBL/Web/Component/Tools/VR/InputForm.pm

```diff
@@ -221,19 +221,18 @@ sub get_cacheable_form_node {
     }]
   });
 
-  if ($current_species eq 'Homo_sapiens') {
-    $input_fieldset->add_field({
+  # Add Mane Select option separately with a different classname for ease of toggling
+  $input_fieldset->add_field({
     'type'        => 'checklist',
-    'field_class' => [qw(_stt_yes _stt_allele _stt_Homo_sapiens)],
```

Comment:
Wonder why this was not working in the first instance. As far as I know this is how we enable a setting for a particular species. Wasn't this working before?

Reply:
@jyothishnt Are you referring to the `field_class` change? It really had nothing to do with why it didn't work, but just thought renaming the `_stt_Homo_sapiens` to `_stt_mane_select` would make it more self descriptive.

Reply:
really? lets discuss tomorrow